astropy/vo/client/conesearch.py

```diff
@@ -81,7 +83,7 @@ def conesearch(center, radius, verb=1, **kwargs):
     Parameters
     ----------
     center : `~astropy.coordinates.SkyCoord`, `~astropy.coordinates.BaseCoordinateFrame`, or sequence of length 2
-        Position of the center of the cone to search.  It may be specified as an
+        Position of the center of the cone to search. It may be specified as an
```

Comment:
I don't like them either but I think the docstring should be consistent. 

If you really, _really_ dislike them then kill the other one too.